devicetypes/smartthings/zwave-motion-light-sensor.src/zwave-motion-light-sensor.groovy

```diff
@@ -89,6 +90,9 @@ def updated() {
 def configure() {
 	// Device wakes up every deviceCheckInterval hours, this interval allows us to miss one wakeup notification before marking offline
 	sendEvent(name: "checkInterval", value: 2 * deviceWakeUpInterval * 60 * 60 + 2 * 60, displayed: false, data: [protocol: "zwave", hubHardwareId: device.hub.hardwareID])
+
+	isEverspring() ?
```

Comment:
I rather see an If statement and then you must have some comment or documentation as to what parameter you are configuring and the reasoning behind that change as well

Reply:
@kianooshST done